docs/src/upgrade.md

```
@@ -0,0 +1,114 @@
+        On the current id=1 sub-category insert line (`INSERT INTO public.forum_sub_categories [...]`), remove the `;` at the end and add the following after it:
+
+        ```sql
+         ON CONFLICT (id) DO UPDATE SET forum_category_id = EXCLUDED.forum_category_id,name = EXCLUDED.name,sort_order = EXCLUDED.sort_order,created_at = EXCLUDED.created_at,created_by_id = EXCLUDED.created_by_id,threads_amount = EXCLUDED.threads_amount,posts_amount = EXCLUDED.posts_amount,forbidden_classes = EXCLUDED.forbidden_classes,new_threads_restricted = EXCLUDED.new_threads_restricted;
+        ```
+
+    7. Be sure that `name` in `public.css_sheets`, `default_css_sheet_name` in `public.arcadia_settings`, and all `css_sheet_name` in `public.users` (for ALL users) is set to `arcadia`. If its not, the restore will not work.  
+    Its because in my instance, I actually change the name of the default css sheet name, and in the default migration (`sqlx migrate run`), the default name is **arcadia**.
+
+6. Restore:
+
+    ```bash
```

markdownlint-cli2 (0.23.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)